tests/pytest_runner.py

```diff
@@ -1,17 +1,29 @@
+import glob
 import os
 
 import pytest
 
-from yt.config import ytcfg
-
 if __name__ == "__main__":
     os.environ["OMP_NUM_THREADS"] = "1"
     pytest_args = [
-        f"--local-dir={os.path.join(ytcfg.get('yt', 'test_data_dir'), 'answers')}",
-        "-c=pytest_answer.ini",
-        "--junitxml=unittests.xml",
-        "--answer-big-data",
-        f"-n {int(os.environ.get('NUM_WORKERS', 6))}",
+        "-s",
+        "-v",
+        "-rsfE",
+        "--with-answer-testing",
+        "-m answer_test",
+        f"-n {int(os.environ.get('NUM_WORKERS', 1))}",
         "--dist=loadscope",
     ]
-    pytest.main(pytest_args)
+    pytest.main(pytest_args + ["--local-dir=answer-store", "--junitxml=answers.xml"])
+
+    if (files := glob.glob("generate_test*.txt")) :
```

Comment:
as glad as I am to see a walrus operator used in yt, it only exists since Python 3.8, so we can't use it until 3.8 is our oldest supported version (right now we're on 3.6)

Reply:
Except this is not a script for a public consumption, nor even a part of yt's API. It's something that's solely being used on testing infra. Jenkins is using py3.8 and I'm not aware of any downgrade plans. GHA answers tests are not even added yet, but I'll make sure they're 3.8+

Reply:
that's what I was missing I guess. Could this be documented in the file itself please ?

Reply:
added a top level comment about the purpose of the file